cabal-testsuite/src/Test/Cabal/Server.hs

```diff
@@ -254,7 +254,8 @@ initServer s0 = do
     pid <- withProcessHandle (serverProcessHandle s0) $ \ph ->
               case ph of
                   OpenHandle x   -> fmap show (Win32.getProcessId x)
-                  ClosedHandle  _ -> return (serverProcessId s0)
+                  -- TODO: handle OpenExtHandle?
+                  _              -> return (serverProcessId s0)
```

Comment:
Looks like the `#ifdef` could be reduced to just the `OpenHandle` line now.

Reply:
oh yeah, good catch